utils/flags/flags.go

```diff
@@ -393,6 +393,12 @@ func GetCliFlags() []cli.Flag {
 			DefaultText: "2",
 			EnvVars:     []string{"BAZEL_REMOTE_S3_KEY_VERSION"},
 		},
+		&cli.IntFlag{
+			Name:        "s3.max_idle_conns",
+			Usage:       "The maximum number of idle connections to use when using the S3 proxy backend.",
+			DefaultText: "1024",
```

Comment:
Is this correct? The link to minio code shows 256 (and 16 per host).

Reply:
Do you think I should preserve the existing defaults? In practice I think they're not good for bazel-remote, which I why I changed it here. 

`MaxIdleConnsPerHost: 16` This is the effective limit in place prior to my change, because we're only working with one host, eg `s3.us-east-1.amazonaws.com`

`MaxIdleConns: 256` This would never be used, since we can't exceed MaxIdleConnsPerHost

I can see the argument for compatibility though, and preserving `MaxIdleConnsPerHost: 16`, but that would also mean adding a second flag for `MaxIdleConns: 256` or setting that value to same as MaxIdleConnsPerHost (16)